docs/docs/rasa-pro-setup.mdx

```diff
@@ -0,0 +1,120 @@
+---
+id: rasa-pro-setup
+sidebar_label: Rasa Pro Setup
+title: Rasa Pro Setup
+description: Installation of Rasa Pro
+abstract: Install Rasa Pro in your production environment to create an enterprise ready Rasa deployment.
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Rasa Pro is compromised of two packages. The packages need to be installed
+separately and in different environments:
+
+- **Rasa Plus**, a drop in replacement for Rasa Open Source for an
+  enterprise deployment. Rasa Plus should be used to deploy your assistant
+  into production.
+- **Rasa Pro Services**, flexible infrastructure and APIs on top of Rasa
+  Open Source. Rasa Pro Services should be deployed alongside, but separately
+  from your production assistant. Rasa Pro Services will connect to your
+  production assistant using Kafka.
+
+## Rasa Plus
+
+:::warning
+I think this content is written currently in the rasa-plus repo.
+:::
+
+## Rasa Pro Services
+
+Installation of the data pipeline requires the deployment of a docker
+container. The container will receive data from a Kafka instance and
+write information into a database.
+
+### Prerequisites
+
+#### Existing Services
+
+- A production deployment of Kafka is required to set up Rasa Pro.
+  We recommend to use [Amazon Managed Streaming for
+  Apache Kafka](https://aws.amazon.com/msk/).
+- A production deployment of a data lake needs to be connected to
+  the data pipeline. Rasa Pro support the following data lakes:
+  1. [Amazon Redshift](https://aws.amazon.com/redshift/) (recommended)
+  2. [PostgreSQL](https://aws.amazon.com/rds/postgresql/) (version >= 11)
+
+#### Hardware requirements
+
+:::warning
+TODO: add once we have done load testing. add minimal and recommended.
+should be aligned with AWS machine types.
+:::
+
+- CPU
+- Disk
+- Memory
+
+### Installation and Configuration
+
+<div align="center">
+  <img
+    alt="An overview over the components of Rasa Pro."
+    src={useBaseUrl("/img/rasa_pro_overview.png")}
+    width="100%"
+  />
+</div>
+
+Rasa Plus streams data to Kafka which gets consumed by Rasa Pro Services.
+The different services need to be run and configured to be able to communicate.
+
+1.  Configure Rasa Plus to stream data to Kafka. The configuration for
+    Kafka should be put in a configuration
+    file `endpoints.yml`:
+
+    ```yaml-rasa title="endpoints.yml"
+    event_broker:
+      type: kafka
+      partition_by_sender: True
+      topic: rasa-events
+      url: <BROKER URL>
+    ```
+
+    The `<BROKER_URL>` needs to be replaced with an address of a bootstrap
+    server of your Kafka cluster. Configuration examples for different
+    security protocols and other parameters can be found in the
+    [Kafka Event Broker](https://rasa.com/docs/rasa/event-brokers/#kafka-event-broker)
+    documentation.
+
+2.  Rasa Pro Services is a docker image. Authenticate to pull
```

Comment:
The auth section is quite an important and lengthy step, including the setting of the license env var.
Since this is duplicated across rasa-plus too, I'd suggest extracting this into a separate page only on Authentication that then can be linked to in both rasa-plus and rasa-pro-services installation pages.

Reply:
sounds good :+1: do we already have a PR that is pulling in the docs from Rasa Plus into these docs here?

Reply:
did we already tackle this?

Reply:
From what I can see, this Google authentication set-up section is still only in Rasa-plus installation page. Would you like to add the auth requirement in prerequisites on the Rasa Pro services deployment page and then link to this auth section?

Reply:
@ancalita thanks for following up 🙏🏻 Tom addressed your comment here https://github.com/RasaHQ/rasa/pull/11591/commits/07d018c89e53d11fc16f03dc255c4a57906ee407

(it's more lightweight, we can always revisit later but it does the job)

Reply:
Yup, that's looks good - however I was more referring to link to the google auth instructions detailed in the installation of rasa-plus, but maybe that's not required here?

Reply:
This is the section that might be too buried and should be linked in rasa pro services section too: https://11591--rasahq-docs-rasa-v2.netlify.app/docs/rasa/installation/rasa-pro/installation#authentication-set-up